### PR TITLE
fix: [ANDROAPP-4237] dashboard on landscape

### PR DESCRIPTION
**Pull request by mmmateos**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://jira.dhis2.org/browse/

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardPageConfigurator.kt [hunk 1]
```diff
@@ -3,11 +3,12 @@ package org.dhis2.usescases.teiDashboard
...
 
 class TeiDashboardPageConfigurator(
-    val dashboardRepository: DashboardRepository
+    val dashboardRepository: DashboardRepository,
+    val isPortrait: Boolean
 ) : NavigationPageConfigurator {
 
     override fun displayDetails(): Boolean {
-        return true
+        return isPortrait
```
Comment (andresmr) [inline app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardPageConfigurator.kt:11]: I don't understand the objective of the function, displayDetails() returns a boolean but is not a question, maybe you could change the name to a more descriptive function name.

File: app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardMobileActivity.java [hunk 2]
```diff
@@ -119,7 +119,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             enrollmentUid = getIntent().getStringExtra(ENROLLMENT_UID);
         }
 
-        ((App) getApplicationContext()).createDashboardComponent(new TeiDashboardModule(this, teiUid, programUid, enrollmentUid)).inject(this);
+        ((App) getApplicationContext()).createDashboardComponent(new TeiDashboardModule(this, teiUid, programUid, enrollmentUid, OrientationUtilsKt.isPortrait())).inject(this);
```
Comment (andresmr) [inline app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardMobileActivity.java:122]: What happens if you rotate the screen once you have injected the component? does this parameter changes?

**Review by andresmr — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardMobileActivity.java (hunk 2 above)
Comment (mmmateos) [reply app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardMobileActivity.java:122]: Yes, when rotation changes this is updated

File: app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardPageConfigurator.kt (hunk 1 above)
Comment (mmmateos) [reply app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardPageConfigurator.kt:11]: This class overrides the NavigationPageConfigurator that have all those methods to handle the bar sections visibility

**Review by JaimeToca — APPROVED** [review]

**Comment by JaimeToca** [comment]
@Balcan or @ferdyrod could you check this PR please? Thanks !!!

**Review by ferdyrod — APPROVED** [review]

**Pull request merged**